reporter/amqp/amqp.go

```diff
@@ -9,10 +9,9 @@ import (
 	"log"
 	"os"
 
-	"github.com/streadway/amqp"
-
 	"github.com/openzipkin/zipkin-go/model"
 	"github.com/openzipkin/zipkin-go/reporter"
+	"github.com/rabbitmq/amqp091-go"
```

Comment:
How about following their suggestion? https://github.com/rabbitmq/amqp091-go#package-name

Reply:
Swapped over to the amqp alias